packages/python-django-filter/python-django-filter.spec

```diff
@@ -71,6 +67,9 @@ set -ex
 
 
 %changelog
+* Tue Dec 12 2023 Patrick Creech <pcreech@redhat.com> - 23.2-4
```

Comment:
I don't think we need to remove this, we are obsoleting this package since python 3.8. This one was not added in the mass deprecation.